yang/frr-bgp-common.yang

```diff
@@ -0,0 +1,1042 @@
+  }
+  grouping med-config {
+    description
+      "Configuration relating to MED";
+    container med-config {
+     leaf enable-med {
+       type boolean;
+       default false;
+       description
+         "Flag to enable receiving of MED metric attribute
+          in routing updates.";
+     }
+
+     leaf max-med-admin {
+       type uint32 {
+         range "0..4294967295";
+       }
+       default 4294967294;
+       description
+         "Tells the router to announce routes with this MED value
+          This MED value is applicable for indefinate time";
+     }
+
+     leaf max-med-onstart-up-time {
+       type uint32 {
+         range "5..86400";
+       }
+       units "seconds";
+       description
+         "Tells the router to announce routes with MED value,
+          This MED value is applicable for this duration during
+          startup";
+     }
+
+     leaf max-med-onstart-up-value {
+       when "../max-med-onstart-up-time != 0";
+       type uint32 {
+         range "0..4294967295";
+       }
+       default 4294967294;
+       description
+         "Tells the router to announce routes with this MED value
+          This MED value is applicable for onstart up time";
+     }
+   }
+  }
+  grouping route-reflector-config {
+    description
+      "grouping used to include route reflector
+       configuration for BGP global";
+    container route-reflector {
+      description
+        "Route reflector parameters for the BGP global";
+      leaf route-reflector-cluster-id {
+        type bt:rr-cluster-id-type;
+        description
+          "Route Reflector cluster id to use when local router is
+           configured as a route reflector.  Commonly set at the
+           group level, but allows a different cluster id to be set
+           for each neighbor.";
+        reference
+          "RFC 4456: BGP Route Reflection: An Alternative to
+           Full Mesh.";
+      }
+      leaf no-client-reflect {
+        type boolean;
+        default "false";
+        description
+          "When set to 'true', this disables route redistribution
+           by the Route Reflector. It is set 'true' when the client is
+           fully meshed to prevent sending of redundant route
+           advertisements.";
+        reference
+          "TODO: Add reference when IETF writes a draft describing
+           this.";
+      }
+      leaf allow-outbound-policy {
+        type boolean;
+        default "false";
+        description
+          "When set to 'true', this allow RR to modify the attributes of the
+           reflected IBGP routes as per the out route-map. It is set 'False'
+           RR will not modify the attributes of the reflected IBGP routes as
+           per the out route-map rules.";
+      }
+    }
+  }
+  grouping global-bgp-config {
+    leaf no-ipv4-unicast-active {
+      type boolean;
+      default "false";
+      description
+        "When set to 'true' it will deactive the ipv4-unicast address family
```

Comment:
`When set to 'TRUE," the ipv4-unicast address family, which is activated by default, is deactivated.`

Reply:
done